templates/ts-apollo-mongodb-backend/.graphqlrc.yml

```diff
@@ -2,6 +2,21 @@
 ## Please review configuration and adjust it for your own project
 schema: ./src/schema/schema.graphql
 extensions:
+  graphback:
+    model: ./model/**/*.graphql
+    crud:
+      create: true
+      update: true
+      findOne: true
+      find: true
+      delete: true
+      subCreate: true
+      subUpdate: true
+      subDelete: true
```

Comment:
We should load the CRUD config from here into `buildGraphbackAPI`, otherwise CRUD config needs to be maintained in multiple places

Reply:
I can get rid of these to remove the bloat. Only added for the `graphback-cli` command